streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesNativeApplicationClient.scala

```diff
@@ -103,4 +109,37 @@ object KubernetesNativeApplicationClient extends KubernetesNativeClientTrait {
     flinkConf.safeSet(DeploymentOptions.TARGET, ExecutionMode.KUBERNETES_NATIVE_APPLICATION.getName)
     super.doTriggerSavepoint(request, flinkConf)
   }
+
+  def convertFlinkDeploymentDef(
```

Comment:
private[this] def convertFlinkDeploymentDef ...

Reply:
thank you